src/parser/content-evaluator-module.ts

```diff
@@ -112,11 +113,28 @@ export class ContentEvaluatorModule implements Module {
     return commentsWithScore;
   }
 
+  /**
+   * Will try to predict the maximum of tokens expected, to a maximum of totalTokenLimit.
+   */
+  _calculateMaxTokens(prompt: string, totalTokenLimit: number = 4096) {
```

Comment:
```suggestion
  _calculateMaxTokens(prompt: string, totalTokenLimit: number = 16384) {
```

> gpt-4o-2024-08-06	Latest snapshot that supports [Structured Outputs](https://platform.openai.com/docs/guides/structured-outputs)	128,000 tokens	16,384 tokens	Up to Oct 2023

> [2] We are releasing this model for developers and researchers to explore OpenAI's latest research. For production use, OpenAI recommends using dated GPT models, which are optimized for API usage.

Model | Description | Context window | Max output tokens | Training data
-- | -- | -- | -- | --
gpt-4o-2024-08-06 | Latest snapshot that supports Structured Outputs | 128,000 tokens | 16,384 tokens | Up to Oct 2023